src/openms/source/METADATA/ProteinIdentification.cpp

```diff
@@ -81,7 +81,8 @@ namespace OpenMS
     enzyme(UNKNOWN_ENZYME),
     missed_cleavages(0),
     peak_mass_tolerance(0.0),
-    precursor_tolerance(0.0)
+    precursor_tolerance(0.0),
+    digestion_enzyme("unknown_enzyme","")
```

Comment:
An 'Enzyme' is HUGE! Our ID datastructures are already too memory intensive. Please don't make it worse.
A string is uniquely identifyable right?!
Better would be an enum, but since the DB is dynamic I guess thats hardly possible.
Maybe a UID (8byte instead of 24+X)...